block-header-gossip.md

```diff
@@ -0,0 +1,198 @@
+# Block header gossip 
+The document describes how the block information (accumulator & block headers) are exchanged between the portal network clients. 
+
+Each client has the responsibilty to track the tip of the chain and to store accumulators and block header information locally. Clients will be sharing the information with requesting clients in the network to maintain the health of the network as a whole
+
+
+## Block storage
+Each client will be storing blocks in two forms. As accumulators where the entire block history is recorded and as partial block headers where only a subset of the block header is stored. Partial block headers are only stored for the most recent N number of blocks.
+
+### Accumulator
+Portal network's accumulator will be based on the [double-batched merkle log accumulator](https://ethresear.ch/t/double-batched-merkle-log-accumulator/571) 
+
+#### Epoch Accumulator
+A fixed sized accumulator of a proposed length of 2048. Block info will be added in into the SSZ list until it has filled up all 2048 of its entries. After which the next block will be included in a new epoch accumulator.
```

Comment:
A common pattern is to use a variable for this:

```
EPOCH_SIZE = 2048

A fixed size accumulator of length `EPOCH_SIZE`...
```

Reply:
I suggest we use the same size as the beacon chain which I believe is `8192`

Reply:
It would be good to have an analysis of total storage sizes that different choices of EPOCH_SIZE will have on clients.  How big do we expect the serialized accumulator to be at various chain lengths?  How fast does it grow?

Reply:
leaving down the results i have here for the time being (stored in txt file)
epoch accumulator with size 2048 - 128kb
master accumulator with results from 6450 epochs  - 202kb

allow me to generate more results by playing around with the epoch size these couple of days